python/composio/tools/local/codeanalysis/constants.py

```diff
@@ -6,4 +6,4 @@
 FQDN_FILE = "fqdn_cache.json"
 DEEPLAKE_FOLDER = "deeplake"
 TREE_SITTER_FOLDER = os.path.join(CODE_MAP_CACHE, "tree_sitter_cache")
-EMBEDDER = "sentence-transformers/all-MiniLM-L6-v2"
+EMBEDDER = "sentence-transformers/all-mpnet-base-v2"
```

Comment:
The change in the EMBEDDER constant from "all-MiniLM-L6-v2" to "all-mpnet-base-v2" might affect performance. Ensure that this change has been thoroughly tested and doesn't negatively impact the system's performance.